gbus/bus.go

```diff
@@ -281,6 +283,23 @@ func (b *DefaultBus) NotifyHealth(health chan error) {
 	b.healthChan = health
 }
 
+//GetHealth implements Health.GetHealth
+func (b *DefaultBus) GetHealth() HealthCard {
+	dbInitiated := b.TxProvider != nil
+	var dbConnected bool
+
+	if dbConnected = false; dbInitiated {
```

Comment:
wouldn't dbConnected always evaluate to false here ? if so please remove from the if check

Reply:
it's  a short way to write this:
```suggestion
	if dbInitiated {
	dbConnected = true 
	} else{
	dbConnected = false
	}
```
I'll change it to be more readable